relay-metrics/src/aggregation.rs

```diff
@@ -1066,21 +1078,23 @@ impl Aggregator {
 
         relay_log::trace!("flushing {} buckets to receiver", buckets.len());
 
-        self.receiver
-            .send(FlushBuckets::new(self.project_key, buckets))
-            .into_actor(self)
-            .and_then(|result, slf, _ctx| {
-                if let Err(buckets) = result {
-                    relay_log::trace!(
-                        "returned {} buckets from receiver, merging back",
-                        buckets.len()
-                    );
-                    slf.merge_all(buckets).ok();
-                }
-                fut::ok(())
-            })
-            .drop_err()
-            .spawn(context);
+        for (project_key, buckets) in buckets.into_iter() {
+            self.receiver
+                .send(FlushBuckets::new(project_key, buckets))
+                .into_actor(self)
+                .and_then(clone!(project_key, |result, slf, _ctx| {
```

Comment:
Can we double-check why this `clone!` is necessary? `ProjectKey` implements `Copy`, so in worst case a `move` closure should do.

Reply:
I think I remember a quirk where Copy-values don't get copied but move the same way as Clone-values into the closure. however even then that shouldn't pose problems because there's no use after the closure and the one extra-use surely would behave like a regular Copy.

Reply:
appears to be unnecessary